experiments/utils.h

```diff
@@ -0,0 +1,96 @@
+#ifndef JOBS_H
+#define JOBS_H
+
+#include <openssl/sha.h>
```

Comment:
I had some issues getting my esp to compile openssl when I was adding the sha256 to fayksic - may want to add to base Readme how to install and

Reply:
I actually never got it to work and used the existing implementation of sha256